index.html

```diff
@@ -33,57 +34,62 @@
                 <h1>Learning version control using Git</h1>
                 <p>
                     <small>
-                        Created by
+                        Created by:
                             <a href="http://www.SanyamKhurana.com">Sanyam Khurana</a> /
                             <a href="http://twitter.com/ErSanyamKhurana">@ErSanyamKhurana</a> /
-                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a>
+                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a> /
+                            <a href="https://thegeekyway.com">The Geeky Way</a><br>
+                        Edited by:
```

Comment:
Move it to contributors slide, including all the people who've contributed.

Reply:
That's already done. Removing this bit.